src/compiler.js

```diff
@@ -56,8 +56,8 @@ module.exports = class GherkinTestcafeCompiler {
     this.tags = getTags();
     this.cucumberExpressionParamRegistry = getParameterTypeRegistry();
     this.externalCompilers = [
-      new TestcafeESNextCompiler(),
-      new TestcafeTypescriptCompiler(compilerOptions[CustomizableCompilers.typescript]),
+      new TestcafeESNextCompiler(compilerOptions[CustomizableCompilers.typescript]),
```

Comment:
I'm going to review this change in more detail because at first glance it seems pretty illogical

Reply:
Thank you for you extensive search and solution!